### PR TITLE
enhancement: Replaced all upper case names

### LINKED ISSUE #250: Change UI to use Capital Letters only for first letter (avoid to capitalize all) and Increase space above Save/Transfer button
1. Please change the UI to use Capital Letters only for first letter (avoid to capitalize all).

2. Increase space above Save/Transfer button in all views. It is currently very close to the above area.

### PR DESCRIPTION
**Pull request by yashk2000**
Fixes #250 

Changes: All existing Upper case strings have been made lower case with only the first letter capital. 

Screenshots for the change:

Screenshot_20190503-232905
Screenshot_20190503-232910

### CODE REVIEW

**Comment by yashk2000** [comment]
@adityastic @mariobehling @iamareebjamal please review.

File: app/src/main/java/com/nilhcem/blenamebadge/data/device/model/Message.kt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package com.nilhcem.blenamebadge.data.device.model
 
-data class Message(val hexStrings: List<String>, val flash: Boolean = false, val marquee: Boolean = false, val speed: Speed = Speed.ONE, val mode: Mode = Mode.LEFT)
\ No newline at end of file
+data class Message(val hexStrings: List<String>, val flash: Boolean = false, val marquee: Boolean = false, val speed: Speed = Speed.ONE, val mode: Mode = Mode.Left)
```
Comment (adityastic) [inline app/src/main/java/com/nilhcem/blenamebadge/data/device/model/Message.kt]: Let this be caps Please

File: app/src/main/java/com/nilhcem/blenamebadge/data/device/model/Mode.kt [hunk 2]
```diff
@@ -4,13 +4,13 @@ import androidx.annotation.StringRes
...
-    LEFT(0x00, R.string.mode_left),
-    RIGHT(0x01, R.string.mode_right),
-    UP(0x02, R.string.mode_up),
-    DOWN(0x03, R.string.mode_down),
-    FIXED(0x04, R.string.mode_fixed),
-    SNOWFLAKE(0x05, R.string.mode_snowflake),
-    PICTURE(0x06, R.string.mode_picture),
-    ANIMATION(0x07, R.string.mode_animation),
-    LASER(0x08, R.string.mode_laser)
+    Left(0x00, R.string.mode_left),
```
Comment (adityastic) [inline app/src/main/java/com/nilhcem/blenamebadge/data/device/model/Mode.kt]: These are constants, should be caps

**Comment by yashk2000** [comment]
@adityastic changes done. Please review.

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Pull request closed without merging**